test/e2e/framework/network_pods.go

```diff
@@ -238,6 +239,17 @@ func (np *NetworkPod) GetLog() string {
 	return out.String()
 }
 
+func securityContext() *v1.SecurityContext {
+	return &v1.SecurityContext{
+		AllowPrivilegeEscalation: pointer.Bool(false),
```

Comment:
This fixes the pod on OCP 4.12:

```suggestion
		AllowPrivilegeEscalation: pointer.Bool(false),
		RunAsNonRoot:             pointer.Bool(true),
		SeccompProfile:           &v1.SeccompProfile{Type: v1.SeccompProfileTypeRuntimeDefault},
```

In addition, the custom pod (`buildCustomPod`) needs to have the same security context.

Reply:
These settings may be [problematic](https://github.com/submariner-io/submariner-operator/issues/2317#issuecomment-1298622098).